specs/ldm/ldm-service.yaml

```diff
@@ -0,0 +1,158 @@
+openapi: 3.0.1
+info:
+  title: "LDM OSLC service"
+  version: “1.0”
+servers:
+  - url: 'http://localhost:8080'
+    description: Generated server url
+paths:
+  /discover-links:
+    post:
+      tags:
+        - link-index-controller
+      operationId: getResources
+      parameters:
+        - name: Configuration-Context
+          in: header
+          required: true
+          schema:
+            type: string
+            format: uri
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/GetIncomingLinks'
+        required: true
+      responses:
+        '200':
+          description: OK
+          content:
+            application/rdf+xml:
+            text/turtle:
+            application/json+ld:
+        '400':
+            description:  Bad request
+
+  /rootservices:
```

Comment:
OSLC has a standardized URL for discovery of the root service document.

Reply:
According to LDM Spec: LDM Service URI is configured explicitly in each LDM client. Therefore no dedicated discovery protocol is needed.